src/VirtualMonoRepo/disallowed-sb-binaries.txt

```diff
@@ -0,0 +1,51 @@
+# arcade
```

Comment:
This file should have a comment at the top that describes what disallowed sb binaries are and how they are treated by the system. (Same for allowed-sb-binaries.txt.)

Reply:
Resolved in https://github.com/dotnet/installer/pull/19043